packages/flutter/lib/src/widgets/app.dart

```diff
@@ -163,6 +164,11 @@ class WidgetsApp extends StatefulWidget {
   /// The list of observers for the [Navigator] created for this app.
   final List<NavigatorObserver> navigatorObservers;
 
+  /// The platform widgets should adapt to target.
+  ///
+  /// Defaults to the current platform.
```

Comment:
Done. Also updated existing comment for [ThemeData] to say
Defaults to [defaultTargetPlatform].